api/s2n.h

```diff
@@ -54,6 +54,7 @@ extern int s2n_config_free_dhparams(struct s2n_config *config);
 extern int s2n_config_free_cert_chain_and_key(struct s2n_config *config);
 extern int s2n_config_set_nanoseconds_since_epoch_callback(struct s2n_config *config, int (*nanoseconds_since_epoch)(void *, uint64_t *), void * data);
 extern const char *s2n_strerror(int error, const char *lang);
+extern const char *s2n_strerror_debug(void);
```

Comment:
Just a thought, but HWYF about using s2n_strerror() but with a special S2N_DEBUG enum value? That way we could re-use the existing interface, with its "lang" parameter and so on.

Reply:
Opted to use `s2n_strerro_debug(int error, const char *lang)` to preserve API compatibility.